docs/advanced-topics/custom-serialization.md

```diff
@@ -3,13 +3,13 @@ id: custom-serialization
 title: Custom Serialization
 ---
 
-When using `RPC`'s, `NetworkVariable`'s or any other MLAPI related task that requires serialization. The MLAPI uses a default serialization pipeline that looks like this:
+When using `RPC`'s, `NetworkVariable`'s or any other Netcode for Gameobjects (Netcode) related task that requires serialization. The Netcode uses a default serialization pipeline that looks like this:
```

Comment:
Netcode for GameObject is capitalized as **N**etcode for **G**ame**O**bjects with O also being capitalized.
I think you'll need to change that in all markdown docs, unfortunately :)